src/Matej.php

```diff
@@ -39,6 +39,13 @@ public function setHttpClient(HttpClient $client): self
         return $this;
     }
 
+    public function setBaseUrl($baseUrl): self
```

Comment:
I'm thinking about either marking this as `@internal` or adding phpdoc, explaining the usecase is mostly for integration testing... But I don't have a strong feeling about it :)